library/std/src/process.rs

```diff
@@ -2019,6 +2019,36 @@ impl Child {
         self.handle.kill()
     }
 
+    /// Ask a child process to exit.
+    ///
+    /// On Unix platforms this sends SIGTERM, which can be blocked, handled, or ignored.
+    ///
+    /// On Windows, and other platforms where there isn't a corrollary to SIGTERM, this just
```

Comment:
Ah shoot. Typo on `corrollary` -> `corollary`

Reply:
Furthermore, after some googling, I think this should probably be "analogous" instead lol.